src/sentry/search/events/fields.py

```diff
@@ -130,7 +130,7 @@ def key_transaction_expression(user_id, organization_id, project_ids):
         PseudoField(
             USER_DISPLAY_ALIAS,
             USER_DISPLAY_ALIAS,
-            expression=["coalesce", ["user.email", "user.username", "user.ip"]],
+            expression=["coalesce", ["user.email", "user.username", "user.ip", "user.id"]],
```

Comment:
I wonder if it makes sense to put `user.id` before `user.ip` since IP may be more likely to change